tests/integration/cluster-users/sar-test.sh

```diff
@@ -232,6 +309,8 @@ function runTests() {
 	echo "--> ${ADMIN_EMAIL} should be able to patch Installation CR in ${NAMESPACE}"
 	testPermissions "patch" "installation" "${NAMESPACE}" "yes"
 
```

Comment:
TBH this `testRafter` function should also contain expected authZ result because IMO admin should have access to CRUD operations on rafter resources. Currently, it's hardcoded for all type of users

Reply:
We want to prevent the creation/update/deletion of any resources by all type of users. Only person who has access to kubeconfig from cluster can create resources.